.github/actions/e2e-testing/edge/action.yml

```diff
@@ -27,27 +31,68 @@ runs:
       uses: GuillaumeFalourd/clone-github-repo-action@main
```

Comment:
Just a note for the previous step: `npm pack` packages up the existing code on the existing branch, so it'll run the code devs are proposing to merge in their PR, rather than whatever code is on `npm` etc. This removes the need for us to merge things before testing them end to end.